content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,142 @@
+number will automatically be assigned to this new RFQ. From here, assign the vendor and order
+deadline, which lets the vendor know when they should confirm the quotation so that Odoo can
+confirm it into a PO.
+
+Additional fields
+~~~~~~~~~~~~~~~~~
+
+   :guilabel:`Vendor reference` points to the sales and delivery order numbers sent by the vendor.
+   This will come in handy once products are received and the PO needs to be matched to the delivery
+   order.
+
+   Under :guilabel:`Agreement`, blanket orders are long-term purchase agreements on recurring orders
+   with set pricing. To view and configure blanket orders, head to
+   :menuselection:`Purchase app --> Orders --> Purchase agreements`.
+
+   :guilabel:`Currency` can be changed if purchasing products from a vendor in another country.
+   To change the main currency or enable additional currencies, head to
+   :menuselection:`Configuration --> Settings --> Currencies`.
+
+   :guilabel:`Deliver to` provides options for the order shipment. Select the receiving warehouse
+   address here, or select :guilabel:`Dropship` to indicate that this order is to be shipped
+   directly to the end customer. When :guilabel:`Dropship` is selected, the
+   :guilabel:`Dropship address` field is enabled. Contact names will populate here from the
+   Contacts app.
+
+.. tip::
+   Expected Arrival is automatically calculated based on the Order Deadline and Vendor Lead Time.
+   Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at delivery.
+
+In the products tab, add the products to be ordered. Select :guilabel:`Add a product` and start
+typing in the product name to select the item from the drop-down menu. :guilabel:`Catalog` can also
+be selected to navigate to a products menu under the chosen vendor. From here, products can be
+added to the cart. If ordering a product that has not been configured, type in the product name,
+select :guilabel:`Product Name` and manually add the unit price. Clicking into the product name
+navigates to its product page, where purchasing details for future orders can be configured.
+
+Send RFQ
+--------
+
+There are a few options to send an RFQ to vendors:
+
+:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email
+address (configured in Contacts). The subject and body can be directly edited from Odoo and the RFQ
+pdf can be edited before sending. Once the email has been sent, the order moves into the
+**RFQ Sent** stage.
+
+:guilabel:`Print RFQ` pulls the order information into a printable document that can be handed to
+the vendor or sent by mail.
+
+Confirm order
+-------------
+
+Clicking :guilabel:`Confirm Order` directly transforms the RFQ into an active order. This can be
+selected once the RFQ has been sent and a confirmation from the vendor has been received, or
+directly from the RFQ stage if there is a prior agreement.
```

Comment:
Suggestion to remove since there are basically no conditions needed to fulfill to click the Confirm Order button. (Except for entering a Vendor and Order Deadline, but you need that for an RFQ too)
```suggestion
Clicking :guilabel:`Confirm Order` directly transforms the RFQ into an active order.
```